pandas/core/generic.py

```diff
@@ -2058,6 +2058,25 @@ def __delitem__(self, key):
         except KeyError:
             pass
 
+    def _take(self, indices, axis=0, convert=True, is_copy=False):
+        """
+        The internal version of `self.take` that will house the `convert`
```

Comment:
more descriptive doc-string with parameters and such

Reply:
Yep, done.